dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CommonWebServerTests20Servlet40.java

```diff
@@ -26,9 +27,14 @@
 @Mode(FULL)
 public class CommonWebServerTests20Servlet40 extends CommonWebServerTests {
 
-    @Test
-    public void testBasicSpringBootApplication20Servlet40() throws Exception {
-        testBasicSpringBootApplication();
+    @After
+    public void stopTestServer() throws Exception {
+        String methodName = testName.getMethodName();
+        if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+            super.stopServer(true, "CWWKT0015W");
+        } else {
+            super.stopServer();
```

Comment:
Why not calling `server.stopServer(true)` here?

Reply:
super.stopServer() will call super.stopServer(true)